### PR TITLE
Fix local anomaly when using normalize_repeats

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

File: bigml/anomaly.py [hunk 1]
```diff
@@ -58,71 +58,111 @@
...
+        h = DEPTH_FACTOR + math.log(population - 1)
+        repeat_depth = max([1.0,
+                            2 * (h - (float(population - 1) / population))])
+    return repeat_depth
+
+
+def build_tree(node, add_population=False):
     """Builds a compressed version of the tree structure as an list of
     lists. Starting from the root node, that is represented by a list:
         [len(children), children1, children2, etc.]
```
Comment (mmerce) [inline bigml/anomaly.py:76]: I need to fix this comment

**Comment by mmerce** [comment]
@unmonoqueteclea, this changes should be used by our current running projects using anomalies. They should only affect corner cases, but still they might help.

File: HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@
 History
 -------
 
+6.1.0 (2021-03-06)
+~~~~~~~~~~~~~~~~~~
+
+- Fixing local anomalies when repeate_normalize has been used.
```
Comment (jaor) [inline HISTORY.rst:9]: typo: repeat

File: bigml/anomaly.py [hunk 3]
```diff
@@ -58,71 +58,122 @@
 PREDICATES_OFFSET = 3
 
 
-def build_tree(children, node=None):
+def get_repeat_depth(population):
+    """
+
+    """
```
Comment (jaor) [inline bigml/anomaly.py:64]: missing docstring?

**Review by jaor — APPROVED** [review]
looks good to me.  just a couple of doc nits.

**Comment by mmerce** [comment]
corrected, let's wait for tests to pass

**Pull request merged**